routing/debug.rst

```diff
@@ -31,6 +31,20 @@ the route name as the command argument:
 
     $ php bin/console debug:router article_show
 
+
+If you want to search a route by name it works too, it will then ask you what route you want to get information for:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:router blo
+    
+      Select one of the following routes to display its information:
+      [0] blog
+      [1] blog_show
+     >
+     
+If you choose ``blog_show`` it will show the route information.
```

Comment:
What about merging this all in the previous code example, to reduce the amount of text/lines?

```rst
You can also get very specific information on a single route by including
the route name as the command argument:

.. code-block:: terminal

    $ php bin/console debug:router article_show
    # ...

    # or use part of the name to search for routes
    $ php bin/console debug:router blo
    
      Select one of the following routes to display its information:
      [0] blog
      [1] blog_show
     >
```

Reply:
updated

Reply:
what sentence do you want to drop, the one explaining or the one in the comments?

Reply:
this one: "If you choose ``blog_show`` it will show the route information."

Reply:
done